tekton/src/tasks/fvt/fvt-core-phase4.yml.j2

```diff
@@ -226,23 +225,7 @@ spec:
         - name: TEST_SUITE
           value: smtp
 
-    # 8. podtemplates
-    # -------------------------------------------------------------------------
```

Comment:
Why is the pod templates test being removed?

Reply:
Merged the podtemplates test into operator maturity